packages/cxl-lumo-styles/scss/global.scss

```diff
@@ -12,6 +12,7 @@ html {
   --cxl-color-black-50pct: hsla(0, 0%, 0%, 0.5);
   --cxl-color-brand-blue: hsla(214, 61%, 25%, 1);
   --cxl-color-light-pink: hsl(353, 73%, 96%, 1);
+  --cxl-color-black: hsl(0, 0%, 0%);
```

Comment:
Do we really don't have pure black color in `--lumo` variables?

Reply:
Unfortunately, I didn't find such a variable. --lumo-shade is dark gray.